workers/unity/Packages/io.improbable.gdk.core/Worker/SerializedMessagesToSend.cs

```diff
@@ -91,6 +91,8 @@ public SerializedMessagesToSend()
 
         public void SerializeFrom(MessagesToSend messages, CommandMetaData commandMetaData)
         {
+            messages.GetSerializedComponentUpdates().CopyTo(updates);
```

Comment:
I guess this is where we need to add the network stats?

Reply:
Added them in the actual sending bit. This will be done for all others in the followup PR.
Its a bit of a waste to iterate over them here just for the netstats, only to iterate again when sending.